WooCommerce/Classes/Authentication/Store Creation/Free Trial/FreeTrialSurveyCoordinator.swift

```diff
@@ -21,7 +21,15 @@ final class FreeTrialSurveyCoordinator: Coordinator {
     }
 
     func start() {
-        let survey = FreeTrialSurveyHostingController(viewModel: .init(source: source))
+        analytics.track(event: .FreeTrialSurvey.surveyDisplayed(source: .freeTrialSurvey24hAfterFreeTrialSubscribed))
+
+        let survey = FreeTrialSurveyHostingController(viewModel: .init(source: source,
+                                                                       onClose: { [weak self] in
+            self?.navigationController.dismiss(animated: true)
+        },
+                                                                       onSubmit: { [weak self] in
+            self?.navigationController.dismiss(animated: true)
+        }))
```

Comment:
Nit: these look identical - should we unify them and use only one closure for both?

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/10328/commits/6c37ac2fd5e03a081d6fcbd4123bb660db8e0227